src/main/java/sirius/biz/protocol/DelegateJournalData.java

```diff
@@ -118,6 +122,21 @@ public void setSilent(boolean silent) {
         this.silent = silent;
     }
 
+    /**
+     * Adds an entry to the journal of the given entity.
+     *
+     * @param targetType the type of the entity under which the entity will be written
+     * @param targetId   the id of the entity under which the entity will be written
```

Comment:
```suggestion
     * @param targetType the type of the entity under which the journal entry will be added
     * @param targetId   the id of the entity under which the journal entry will be added
```
Maybe make the wording here less confusing 😅